src/style.css

```
@@ -1 +1,13 @@
-/* styles go here */
+
+.input__container-item {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.input__container {
+  margin-bottom: 20px;
+}
+
+.input__container > label:last-child {
```

```suggestion
.input__container > input__container-item:last-child {
```
use classes to add styles

---

corrected